From c80495a6437247f13c68d50489f6b8d6319ec277 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:26 -0400
Subject: [redhat] kabi: Adding symbol param_get_uint (kernel/params.c)

Message-id: <469cd2f59ac8ca9ae9b9dd4a1edff997188114a7.1410510773.git.poros@redhat.com>
Patchwork-id: 96035
O-Subject: [PATCH RHEL7.1 146/233] kabi: Adding symbol param_get_uint (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type uint.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_get_uint
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_get_uint
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_get_uint

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_get_uint b/redhat/kabi/kabi-module/kabi_ppc64/param_get_uint
new file mode 100644
index 0000000..a64340a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_get_uint
@@ -0,0 +1,2 @@
+#1-
+0xe200d2d5 param_get_uint vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_get_uint b/redhat/kabi/kabi-module/kabi_s390x/param_get_uint
new file mode 100644
index 0000000..a64340a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_get_uint
@@ -0,0 +1,2 @@
+#1-
+0xe200d2d5 param_get_uint vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_get_uint b/redhat/kabi/kabi-module/kabi_x86_64/param_get_uint
new file mode 100644
index 0000000..a64340a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_get_uint
@@ -0,0 +1,2 @@
+#1-
+0xe200d2d5 param_get_uint vmlinux EXPORT_SYMBOL
-- 
1.7.1